templates/docker-compose/setup.sh

```diff
@@ -208,15 +212,15 @@ configure() {
   fi
 
   if [[ -n "${email// }" ]]; then
-    sed -i -E "s/^(MAILTO=).*/\1$(escape_re "${email// }")/" .env
+    sed -i "" -e "s/^\(MAILTO=\).*/\1$(escape_re "${email// }")/" .env
```

Comment:
`sed -i ""` seems to be unique to sed posix / macos

we'll need to do https://stackoverflow.com/a/57766728